### DISCUSSION TITLE
🔗 Extend import/export kit to include Elementor setting (features, fonts etc...)

### DISCUSSION THREAD

**Discussion by Mulli** [elementor/elementor#28325] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

The goal is to export a complete "set" of Elementor templates (>10), setting etc.. so that it will be integrated into a live site.

The import/export process *ignores* Elementor setting such as:
1. Use/dont use global fonts, colors settings
2. "Features" - selections may differ betwenn source and target

### Describe the solution you'd like.

My request is to NOTIFY and ALLOW the importer to adopt (or not) the source setting
as part of the process of comparing used plugins, current post types, etc...

Bottom line is to have *reliable* import/export that does not break the site
Currently the process is incomplete. One may regard it as a bug...

### Describe alternatives you've considered.

I spent a lot of time finding out that things literaly break! (e.g. nested elements active on source by not on target)
Current work around is to check all settings manually. grrrr.....

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.
